v3/src/components/graph/adornments/movable-value/movable-value.tsx

```diff
@@ -198,11 +198,18 @@ export const MovableValue = observer(function MovableValue (props: IProps) {
   // Refresh the value when the axis changes
   useEffect(function refreshAxisChange() {
     return autorun(() => {
+      // We observe changes to the axis domains within the autorun by extracting them from the axes below.
+      // We do this instead of including domains in the useEffect dependency array to prevent domain changes
+      // from triggering a reinstall of the autorun.
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { domain: xDomain } = xAxis
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { domain: yDomain } = yAxis
```

Comment:
FYI, you can also use:
```suggestion
      const { domain: xDomain } = xAxis // eslint-disable-line @typescript-eslint/no-unused-vars
      const { domain: yDomain } = yAxis // eslint-disable-line @typescript-eslint/no-unused-vars
```

Reply:
That's a bit cleaner. Thanks 👍